R/msnset2se.R

```diff
@@ -0,0 +1,48 @@
+#' SummarizedExperiment to MSnSet object conversion
+#'
+#' \code{se2msnset} generates an MSnSet object from an SummarizedExperiment object.
+#'
+#' @param se SummarizedExperiment,
+#' Object which will be turned into an MSnSet object.
+#' @return MSnSet object.
+#' @export
+se2msnset <- function(se) {
+  # Check input
+  assertthat::assert_that(inherits(se, "SummarizedExperiment"))
```

Comment:
Currently we don't use `assertthat` for evaluation of input variables. While it is very convenient it don't think we are accepting a new dependency for just an easy `if (inherits(se, "SummarizedExperiment"))` check. BTW: `assertthat` is not listed in the `DESCRIPTION` file.